lib/tasks/screenshots.rake

```diff
@@ -41,7 +41,7 @@ if Rails.env.development?
     screenshot_file = 'about_us.png'
     page.save_screenshot(screenshot_file)
     puts "Saved screenshot #{screenshot_file}"
-    spectre.submit_test('About Us', 'Phantom', 'OSX', 1024, File.new(screenshot_file, 'rb'))
+    spectre.submit_test('About Us', 'Phantom', 'OSX', 1024, File.new(screenshot_file, 'rb'), "", "90%")
```

Comment:
No idea how optional args can be passed, but could this be rewritten as `(project: 'About us', fuzz: '90%')` etc, rather than having to pass the blank `url` arg?